googleservices/cloudmonitoring/client.py

```diff
@@ -28,29 +28,33 @@ def write_timeseries_custon_instance_metric_double_value(self, project_id, metri
         unique_machine_id = get_gce_unique_id(http)
 
         timeseries_dict = {
-            'kind': 'cloudmonitoring#writeTimeseriesRequest',
-            'commonLabels': {
-                'compute.googleapis.com/resource_type': 'instance',
-                'compute.googleapis.com/resource_id': unique_machine_id
-            },
-            'timeseries': [
-                {
-                    'timeseriesDesc': {
-                        'metric': 'custom.cloudmonitoring.googleapis.com/custom/%s' % metric_name,
-                        'labels': {
-                        }
+            'timeSeries': [{
+                'metric': {
+                    'type': 'custom.googleapis.com/custom/%s' % metric_name,
+                    'labels': {},
+                },
+                'resource': {
+                    'type': 'gce_instance',
+                    'labels': {
+                        'instance_id': unique_machine_id,
+                        'zone': 'us-central1-a'}
```

Comment:
I would probably add the zone as a parameter to this function, we might have machines from different zones